centreon/src/Core/Resources/Application/UseCase/FindResourcesByParent/FindResourcesByParentFactory.php

```diff
@@ -47,7 +47,7 @@ public static function createResponse(array $parents, array $children): FindReso
             $found = self::findChildrenAmongResponse($parent->id, $children);
             $resources[] = new ResourcesByParentResponseDto(
                 $parent,
-                $found,
+                array_values($found),
```

Comment:
It doesn't make sense, $found is already a list of objects array<int, ResourceResponseDto>